src/elementary/NucleusID/src/register.hpp

```diff
@@ -1,2 +1,3 @@
 // register the regex
-const std::regex NucleusID::regex{ "^([a-z]{1,2}?\\d{1,3})(_e(\\d+))?$" };
+const std::regex
+NucleusID::regex{ "^([a-z]{1,2}?\\d{1,3})((_e(\\d+))|(\\[continuum\\]))?$" };
```

Comment:
I think @whaeck is having fun with regular expressions

Reply:
It makes things sooooooo much easier :-)